tests/Types.php

```diff
@@ -26,13 +26,30 @@
 /**
  * Await
  */
-assertType('bool', $bridge->await(run(static function (): bool {
-    return true;
+assertType('null', $bridge->await(run(static function (): void {
```

Comment:
This assertion is misplaced here too.

Same as https://github.com/reactphp-parallel/infinite-pool/pull/60#discussion_r1910057904

You cannot take the value of a `void` expression, as it is has no possible value.

You can turn it into an expression by having a closure around it:

```diff
-assertType('null', $bridge->await(run(static function (): void {
+assertType('Closure(): void', (fn () => $bridge->await(run(static function (): void {
```